test/09_platform_selector_problem/spam.c

```diff
@@ -0,0 +1,57 @@
+#include <Python.h>
+#if defined(__linux__)
+#include <malloc.h>
+#endif
+
+static PyObject *
+spam_system(PyObject *self, PyObject *args)
+{
+    const char *command;
+    int sts = 0;
+
+    if (!PyArg_ParseTuple(args, "s", &command))
+        return NULL;
+
+#if defined(__linux__)
+    sts = malloc_info(0, stdout);
+#endif
```

Comment:
We don't need this in this test; this was just to test `manylinux2010`-only builds. Could you just copy the `01_basic` version?

Reply:
Done.